core/controllers/admin.py

```diff
@@ -848,3 +848,13 @@ def post(self):
             self.render_json({})
         else:
             raise self.InvalidInputException('This app cannot send emails.')
+
+
+class GetInteractionsByExplorationId(base.BaseHandler):
+    """This function returns a list of interaction id's for a exploration id."""
+
+    @acl_decorators.can_access_admin_page
+    def get(self):
+        exploartion_id = self.request.get('exploration_id')
+        exploration = exp_fetchers.get_exploration_by_id(exploartion_id)
+        self.render_json({'title': exploration.title})
```

Comment:
We are expected to have the list of interaction here, right?

@aggarwalpulkit596, is it a WIP PR (and you need any help/advice)? If yes, then I would suggest closing this PR and create a PR on your fork branch and ask anyone gitter for help (with PR link). 
(see guidelines [here](https://github.com/oppia/oppia/wiki/WIP-PRs).)

Reply:
yes i will change this was trying to understand the workflow first now as i am to understand the flow i can easily get the list of interaction from exploration.states.interaction

Reply:
I'm closing this PR till then, feel free to reopen it once it's ready for review. :)